client/my-sites/plans/current-plan/trials/trial-current-plan.tsx

```diff
@@ -27,18 +31,28 @@ const TrialCurrentPlan = () => {
 		setShowAllTrialFeaturesInMobileView( true );
 	};
 
+	const currentPlanSlug = selectedSite?.plan?.product_slug ?? '';
+	const isEcommerceTrial = currentPlanSlug === PLAN_ECOMMERCE_TRIAL_MONTHLY;
+	// const isMigrationTrial = currentPlanSlug === PLAN_MIGRATION_TRIAL_MONTHLY;
+
 	const isMobile = useMediaQuery( '(max-width: 480px)' );
 	const displayAllIncluded = ! isMobile || showAllTrialFeaturesInMobileView;
+	const bodyClass = isEcommerceTrial
+		? [ 'is-ecommerce-trial-plan' ]
+		: [ 'is-migration-trial-plan' ];
```

Comment:
Consider of future hosting trial, should we make this as a `is-bussiness-trial-plan` instead?

Reply:
Sure, good idea.